docs/source/API/core/view/create_mirror.md

```diff
@@ -125,6 +149,21 @@ ImplMirrorType create_mirror_view_and_copy(Space const& space, ViewType const&);
   * `Space`: a class meeting the requirements of [`ExecutionSpaceConcept`](ExecutionSpaceConcept) or [`MemorySpaceConcept`](MemorySpaceConcept)
   * `ImplMirrorType`: an implementation defined specialization of `Kokkos::View`.
 
+* ```c++
+  template <class ViewType, class... ViewCtorArgs>
+  auto create_mirror_view(Impl::ViewCtorProp<ViewCtorArgs...> const& arg_prop,
+                          ViewType const& src);
+  ```
+  If the [`View`](view) constructor arguments `arg_prop` include a memory space and the memory space doesn't match the memory space of `src`, creates a new [`View`](view) in the specified memory_space.
+  If the `arg_prop` don't include a memory space and the memory space of `src` is not host-accessible, creates a new host-accessible [`View`](view).
+  Otherwise, `src` is returned.
```

Comment:
```suggestion
  If the `arg_prop` don't include a memory space and the memory space of `src` is not host-accessible, a new host-accessible [`View`](view) is created.
  Otherwise, `src` is returned.
```